gunicorn.py

```diff
@@ -12,6 +12,6 @@
 
 def on_starting(server):
     """Gunicorn server hook."""
-    with models.db:
-        models.db.execute_sql("CREATE SCHEMA IF NOT EXISTS embedding;")
-        models.db.create_tables(models.MODELS, safe=True)
+    # Perform migrations
+    with models.db.connection_context():
+        models.run_migration()
```

Comment:
Normally it's not considered a good idea to run migration on startup, at least in production. 
It is considered a better strategy to explicitly run them at deployment (or project setup for dev).